.gitignore

```diff
@@ -26,5 +37,4 @@ screensaver.rsxs.*/addon.xml
 # clion
 .idea/
 
-# KDev
-*.kdev4
\ No newline at end of file
+*.patch
```

Comment:
Ignoring patch files makes workings with depends complicated as you have to use `git add -f`.